packages/components/src/components/bal-navigation/bal-navigation-popover/bal-navigation-popover.tsx

```diff
@@ -40,9 +41,25 @@ export class NavigationPopover implements ComponentInterface {
    */
   @Prop() activeColor: Props.BalButtonColor = 'primary'
 
+  /**
+   * Defines the heading of the popover
+   */
+  @Prop() heading?: string
+
+  /**
+   * If 'false', the closing button is not rendered
+   */
+  @Prop() isClosable = true
```

Comment:
- [ ] just closable no is-
- [ ] and build again the components.d.ts is still old